ZipBuilder/Sources/ZipBuilder/FileManager+Utils.swift

```diff
@@ -107,6 +101,9 @@ public extension FileManager {
     }
   }
 
+  // Enable a single unique temporary workspace per execution.
+  static var uniq: String?
```

Comment:
You can actually instantiate this right here and use `let` - `static let unique: String = UUID().uuidString`

Also, I think having the extra `ue` is worth it from a readability standpoint

Reply:
"Static properties may only be declared on a type". Other comments will be addressed in next push.

Reply:
Hmm, I had this in a Playground and it worked as expected:

```
extension FileManager {
  static let unique: String = UUID().uuidString
}

print(FileManager.unique)
print(FileManager.unique)
print(FileManager.unique)
```

Reply:
Done.

I misunderstood and thought you were asking to initialize inside the func. This is much cleaner. Thanks!